include/core/parameters_lagrangian.h

```diff
@@ -178,33 +178,40 @@ namespace Parameters
       // Insertion frequency
       int insertion_frequency;
 
-      // Direction (axis) of insertion of particles (1st, 2nd, 3rd)
-      std::vector<unsigned int> direction_sequence;
+      /* Clearing box: */
```

Comment:
We actually don't use different types of comments than //. I am not sure if this is really necessary, but it is your call here.

Reply:
This way, it is easier to find which set of parameters is used in which method.